apps/paf/behaviours/get-section.js

```diff
@@ -0,0 +1,127 @@
+module.exports = superclass => class extends superclass {
```

Comment:
Is there a reason you used the word set rather than get? Is this file setting the steps and backlinks based on the new navigation design? I wonder if it's worth renaming the behaviour to something like `set-navigation-section`

Reply:
Yeah, you're right;  it is setting properties based on the navigation rather than getting the actual section so I will amend the name

Reply:
Great, now it makes more sense to me 🙂